hack/package/build.sh

```diff
@@ -0,0 +1,139 @@
+#!/usr/bin/env sh
+
+set -e
+# This script is to build pouch rpm package as follows,
+# Following the below command to build rpm
+# 1. Build pouch:rpm image
+#	cd hack/package/rpm
+#	docker build -t pouch:rpm .
+# 2. Mount a directory which contains gpg keys, eg
+#    $ tree rpm/keys/
+#    rpm/keys/
+#    ├── gpg
+#    └── secretkey
+#
+# 3. Run the following command, and enter your pass phrase to sign rpm package
+# docker run -it -v /root/rpm/:/root/rpm pouch:rpm bash -c hack/package/build.sh
+#
+# 4. In this example rpm package will be output in '/root/rpm/package/' directory
+
+DIR="$( cd "$( dirname "$0" )/.." && pwd )"
+
+TMP=$(mktemp -d /tmp/pouch.XXXXXX)
+# lxcfs stable branch
+LXC_TAG='stable-2.0'
+LXC_DIR=$TMP/lxc
+
+KEYDIR=/root/rpm
+PACKAGEDIR=/root/rpm/package
+
+# the branch to build pouch
+POUCH_BRANCH='0.1.x'
```

Comment:
Do we need to change this variable to be : POUCH_BRANCH={$POUCH_BRANCH:?"Must specify pouch branch"} ? the same for VERSION

Reply:
Good idea，I‘ll parse these parameters via `env`